src/common/legacy/src/transformations/convert_opset1_to_legacy/convert_normalizel2_to_normalize_ie.cpp

```diff
@@ -56,7 +56,7 @@ ngraph::pass::ConvertNormalizeL2WithMulToNormalizeIE::ConvertNormalizeL2WithMulT
 
         auto axis = const_axis->cast_vector<size_t>();
         bool across_spatial = !(axis.size() == 1 && axis[0] == 1);
-        bool channel_shared = (constant->get_shape().size() == 1);
+        bool channel_shared = (shape_size(constant->get_shape()) == 1);
```

Comment:
`constant->get_shape().size() == 1` and `shape_size(constant->get_shape()) == 1` aren't equivalent. Why are we have to change this line?